### PR TITLE
DDP-7392 

### PR DESCRIPTION
**Pull request by leonidrakitin**
https://broadinstitute.atlassian.net/browse/DDP-7392

### CODE REVIEW

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoAboutChildV2.java [hunk 1]
```diff
@@ -13,50 +12,55 @@
...
 import java.time.Instant;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
  * One-off task to add adhoc symptom message to TestBoston in deployed environments.
  */
-@Slf4j
```
Comment (Trimax) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoAboutChildV2.java:38]: Why did you remove Lombok's annotation? We're now in the middle of the migration from injection of the logger like this `private static final Logger LOG = LoggerFactory.getLogger(OsteoAboutChildV2.class);` to using Lombok.

**Review by Trimax — CHANGES_REQUESTED** [review]

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoAboutChildV2.java (hunk 1 above)
Comment (leonidrakitin) [reply pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoAboutChildV2.java:38]: my bad, accidentally deleted during a merge.

File: pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoAboutChildV2.java [hunk 2]
```diff
@@ -13,50 +13,56 @@
...
 import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
  * One-off task to add adhoc symptom message to TestBoston in deployed environments.
  */
 @Slf4j
 public class OsteoAboutChildV2 implements CustomTask {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OsteoAboutChildV2.class);
```
Comment (Trimax) [inline pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/OsteoAboutChildV2.java]: Please remove this line of the code.

I guess I have to put some details here. Slf4j annotation above the class says Lombok to inject logger instance automatically. So if you have your class annotated as Slf4j you can freely use logger without a direct definition. Like this: `log.info("Question ('CHILD_HISPANIC') successfully disabled");`

https://projectlombok.org/features/log

**Review by archilsharashenidzebroad — APPROVED** [review]
LGTM

**Review by archilsharashenidzebroad — APPROVED** [review]
LGTM

**Pull request closed without merging**